### ISSUE TITLE
Train xPts model

### ISSUE THREAD

**Issue by hkenawi** [hkenawi/FPL-optimizer#5]
This includes the following:
1. Train the model on the cleaned training data
2. Ensure there are no problems with the vanishing gradient
3. Evaluate the model using a selected error metric on the testing set (MSE, RMSE, MASE, MAPE)
